extensionsGallery-insider.json

```diff
@@ -3481,7 +3481,7 @@
 						}
 					],
 					"statistics": [],
-					"flags": "preview"
+					"flags": ""
```

Comment:
Don't update insiders here. There should be two separate PRs - one targeting `release/extensions` that updates insiders (since that has the GA version now) and one targeting rc1 that has the stable gallery update.

Reply:
Sent a PR for the same.